examples/chip-tool/commands/clusters/ComplexArgument.h

```diff
@@ -195,7 +196,7 @@ class ComplexArgumentParser
 
         size_t size = strlen(value.asCString());
         auto buffer = static_cast<char *>(chip::Platform::MemoryCalloc(size, sizeof(char)));
-        strncpy(buffer, value.asCString(), size);
+        chip::Platform::CopyString(buffer, size, value.asCString());
```

Comment:
This is causing a problem since strncpy does not null terminate while CopyString does